scipy/stats/_stats_py.py

```diff
@@ -8567,6 +8567,10 @@ def energy_distance(u_values, v_values, u_weights=None, v_weights=None):
     independent random variables whose probability distribution is :math:`u`
     (resp. :math:`v`).
 
+    Sometimes the square of this quantity is referred to as the "energy
+    distance" (e.g. in [2]_, [4]_), but as noted in [1]_ and [3]_, this square
+    root version satisfies the axioms of a distance function (metric).
```

Comment:
```suggestion
    distance" (e.g. in [2]_, [4]_), but as noted in [1]_ and [3]_, only the
    definition above satisfies the axioms of a distance function (metric).
```
I think this is needed if we remove "square root" from the previous sentence.

Reply:
Sure.

Reply:
If you're happy with it, care to commit this and merge @rkern? I could call someone else in if it's important, but I avoid self-merging just so it's clear in the commit history that the work was reviewed.